config/runtime.exs

```diff
@@ -144,6 +144,7 @@ if config_env() == :prod do
     end
 
   config :admin, base_host: base_host
+  config :admin, backend_host: base_host
```

Comment:
I'm not really sure here. 
In local I used `localhost:3000` that automatically redirects to `/api`. 
Using `api.graasp.org` in prod would work as it redirects to `/api` too, but using `graasp.org` would require `/api` (so it's not a host anymore).